lib/index.js

```diff
@@ -62,4 +62,4 @@ const fixPackageModule = {
   }
 };
 
-exports.module = fixPackageModule;
+module.exports = fixPackageModule;
```

Comment:
and this was a reason of not correct require

Reply:
Well, glad to listen that it can be useful :). Thanks for fix. 

Ill merge PR. Hope you like it